crates/ruma-client-api/src/r0/backup.rs

```diff
@@ -69,8 +70,26 @@ pub struct KeyBackupData {
     pub session_data: SessionData,
 }
 
+impl KeyBackupData {
+    /// Create a `KeyBackupData` struct.
```

Comment:
Please use the same wording as for other constructors (mentioning the parameters, `Create a *new*`).

Reply:
Actually with one param being a bool (whose purpose won't be very clear in the arg list) and two being the same type, an `Init` type would be better.